flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
any specific reason? Mono works on aarch64

Reply:
I would be interested to try it some time. I know FieldWorks bundles a specific gecko for rendering html, and I don't think that includes aarch64 at present.